protocol/messenger.go

```diff
@@ -74,6 +74,7 @@ var messageCacheIntervalMs uint64 = 1000 * 60 * 60 * 48
 // because installations are managed by the user.
 // Similarly, it needs to expose an interface to manage
 // mailservers because they can also be managed by the user.
+// TODO we may want to change the maps into sync.Maps if we start getting unexpected locks or weird collision bugs
```

Comment:
Ah, this can be removed now I suppose. Do you want me to update my branch?

Reply:
I'll do it in this branch, thanks